api/base/schemas/utils.py

```diff
@@ -29,10 +29,11 @@ def validate_dates(history):
     if history.get('start_date'):
         start_date = datetime.strptime(history['start_date'], '%Y-%m-%d')
 
-    if not history.get('ongoing', False):
-        if history.get('end_date'):
-            end_date = datetime.strptime(history['end_date'], '%Y-%m-%d')
-
-        if history.get('start_date') and history.get('end_date'):
-            if (end_date - start_date).days <= 0:
-                raise InvalidModelValueError(detail='End date must be greater than or equal to the start date.')
+        if not history.get('ongoing', False):
+            if history.get('end_date'):
+                end_date = datetime.strptime(history['end_date'], '%Y-%m-%d')
+                if (end_date - start_date).days <= 0:
+                    raise InvalidModelValueError(detail='End date must be greater than or equal to the start date.')
```

Comment:
Logic and error message don't line up. If you enter the same date for end date and start date, that error is thrown.

Reply:
Changed the if statement to just be less than so if the End Date is equal to or greater than the Start Date, no error will be raised.